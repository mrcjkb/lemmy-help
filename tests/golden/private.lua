local U = {}

---@private
---This is a private function which is exported
---But not considered as part of the API
function U.private()
    print('I am private!')
end

---Only this will be documented
function U.ok()
    print('Ok! I am exported')
end

---@protected
function U.no_emmy()
    print('Protected func with no emmylua!')
end

return U
