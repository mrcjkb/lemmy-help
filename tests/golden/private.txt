U.ok()                                                                    *U.ok*
    Only this will be documented



